thirdparty/optional-lite-3.2.0/LICENSE

```diff
@@ -0,0 +1,23 @@
+Boost Software License - Version 1.0 - August 17th, 2003
```

Comment:
The filename was `LICENSE.txt` in the original. I think we shouldn't rename things without a good reason.